src/app/components/import/ImportModal.js

```diff
@@ -53,12 +54,13 @@ const ImportModal = ({ onClose, ...rest }) => {
     };
 
     const handleAttach = ({ target }) => {
-        const attachedFile = [...target.files].filter(({ type }) => ['text/vcard', 'text/csv'].includes(type))[0];
+        const ext = splitExtension(target.files[0].name)[1];
```

Comment:
I prefer to have verbose variable, use `extension` instead of `ext` which can be confusing with `external`.

Reply:
yep, I was actually not too happy myself with the choice...